src/pudl/etl.py

```diff
@@ -319,9 +319,9 @@ def etl_epacems(
             compression="snappy",
             version="2.6",
         ) as pqwriter:
-            for year, state in itertools.product(
-                epacems_settings.years, epacems_settings.states
-            ):
+            for part in epacems_settings.partitions:
+                year = part["year"]
+                state = part["state"]
```

Comment:
Since I created this `itertools.product` within the settings, I use it instead of making it again from the raw componenets